src/NServiceBus.AcceptanceTests/Recoverability/When_transactional_message_is_moved_to_error_queue.cs

```diff
@@ -0,0 +1,147 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using EndpointTemplates;
+    using NServiceBus.Pipeline;
+    using NUnit.Framework;
+
+    public class When_transactional_message_is_moved_to_error_queue : NServiceBusAcceptanceTest
+    {
+        [TestCase(TransportTransactionMode.SendsAtomicWithReceive)]
+        [TestCase(TransportTransactionMode.TransactionScope)]
+        public async Task Should_not_dispatch_outgoing_messages(TransportTransactionMode transactionMode)
+        {
+            Requires.DtcSupport();
```

Comment:
we should probably split that test further so it can run on transports implementing cross-q-transactions?